src/Components/Home/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "../ArticleCard";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-app-ftk2.onrender.com/api/articles")
+      .then((res) => {
+        setArticles(res.data.articles[0]);
+      });
+  }, []);
+
```

Comment:
It would be great to see a loading pattern here. Your api will spin down after 15 mins of inactivity and take some time to spin back up. While that is spinning up a user would ideally want to know things are working 💡